vortex-datafusion/src/convert/scalars.rs

```diff
@@ -239,3 +239,360 @@ impl FromDataFusion<ScalarValue> for Scalar {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use datafusion_common::ScalarValue;
+    use datafusion_common::arrow::datatypes::i256 as arrow_i256;
+    use rstest::rstest;
+    use vortex::buffer::ByteBuffer;
+    use vortex::dtype::{DType, DecimalDType, Nullability, PType};
+    use vortex::scalar::{DecimalValue, Scalar, i256};
+
+    use super::*;
+
+    #[rstest]
+    #[case::u8_some(Scalar::from(42u8), ScalarValue::UInt8(Some(42)))]
```

Comment:
oh fun, i didn't know you could name the cases this way

Reply:
mysterious are the ways of rstest